lib/screens/events/cubit/events_cubit.dart

```diff
@@ -4,16 +4,17 @@ import 'package:ecellapp/core/res/strings.dart';
 import 'package:ecellapp/core/utils/logger.dart';
 import 'package:ecellapp/screens/events/events_repository.dart';
 import 'package:equatable/equatable.dart';
+import 'package:ecellapp/models/events.dart';
```

Comment:
model file should be called `event.dart`, and class should be called `Event`. It stores details of one event, not multiple events.
```suggestion
import 'package:ecellapp/models/event.dart';
```

Reply:
Done